locations/spiders/campingworld.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import scrapy
+from scrapy.http import JsonRequest, Request
+
+import base64
+
+from locations.items import Feature
+from locations.user_agents import BROWSER_DEFAULT
+
+
+class CampingWorldSpider(scrapy.Spider):
+    name = "campingworld"
```

Comment:
If this is US only, we'd prefer to flag that:
```suggestion
class CampingWorldUSSpider(scrapy.Spider):
    name = "camping_world_us"
```

And a filename of `camping_world_us.py`

Reply:
Updated!